api/src/main/java/io/serverlessworkflow/api/validation/ValidationError.java

```diff
@@ -43,4 +45,18 @@ public void setType(String type) {
   public String toString() {
     return String.format(MSG_FORMAT, type, message);
   }
+
+  @Override
+  public int hashCode() {
```

Comment:
is hashcode and equals override here needed or just cosmetic?

Reply:
This is changed to avoid duplicates when inserting validation errors into a Set implementation

Reply:
do you have a test that makes sure this is done correctly? what if you have multiple workflow states in definition
and both produce the same error? trying to understand why duplication is a bad thing

Reply:
Its pretty trivial IDE generated code, but I will add the tests

Reply:
Added